azure/durable_functions/models/DurableOrchestrationContext.py

```diff
@@ -229,17 +230,22 @@ def call_http(self, method: str, uri: str, content: Optional[str] = None,
             The HTTP request headers.
         token_source: TokenSource
             The source of OAuth token to add to the request.
+        is_raw_str: bool, optional
+            If True, send content as-is.
+            If False (default), serialize content to JSON.
 
         Returns
         -------
         Task
             The durable HTTP request to schedule.
         """
         json_content: Optional[str] = None
-        if content and content is not isinstance(content, str):
-            json_content = json.dumps(content)
-        else:
-            json_content = content
+        if content is not None:
+            if isinstance(content, str) and is_raw_str:
+                # don't serialize the str value - use it as the raw HTTP request payload
+                json_content = content
+            else:
+                json_content = json.dumps(content)
```

Comment:
nit - let's validate that that `is_raw_str` is only used for `str` types. That'll help control the number of valid uses of the API

```suggestion
        # validate parameters
        if (not isinstance(content, str)) and is_raw_str:
            raise TypeError("Invalid use of 'is_raw_str' parameter: 'is_raw_str' is set to 'True' but 'content' is not an instance of type 'str'. Either set 'is_raw_str' to 'False', or ensure your 'content' is of type 'str'.")

        # serialize
        if content is not None:
            if isinstance(content, str) and is_raw_str:
                json_content = content
            else:
                json_content = json.dumps(content)
```